pgrx-examples/decoder/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "decoder"
```

Comment:
oh, only last thought is that it should probably be named `wal_decoder` instead? likelihood of another `decoder` example appearing over sufficient time is approximately 1

Reply:
agreed.

Maybe wal2json_decoder ? There may be others types of WAL decoder in the future ?